Please fix the build:
```
make[1]: Entering directory '/home/circleci/project/exporter/logzioexporter'
make[1]: *** No rule to make target 'dep'.  Stop.
make[1]: Leaving directory '/home/circleci/project/exporter/logzioexporter'
Makefile:91: recipe for target 'for-all-target-./exporter/logzioexporter' failed
make: *** [for-all-target-./exporter/logzioexporter] Error 2
```

--- a/exporter/logzioexporter/Makefile
+++ b/exporter/logzioexporter/Makefile
@@ -1,0 +1,1 @@
+include ../../Makefile.Common

--- a/exporter/logzioexporter/config.go
+++ b/exporter/logzioexporter/config.go
@@ -15,8 +15,6 @@
 package logzioexporter
 
 import (
-	"errors"
-
 	"go.opentelemetry.io/collector/config/configmodels"
 )
 
@@ -26,11 +24,4 @@ type Config struct {
 	Token                         string `mapstructure:"account_token"`           // Your Logz.io Account Token, can be found at https://app.logz.io/#/dashboard/settings/general
 	Region                        string `mapstructure:"region"`                  // Your Logz.io 2-letter region code, can be found at https://docs.logz.io/user-guide/accounts/account-region.html#available-regions
 	CustomEndpoint                string `mapstructure:"custom_endpoint"` 		  // Custom endpoint to ship traces to. Use only for dev and tests. The will override the Region parameter
-}
-
-func (c *Config) validate() error {
-	if c.Token == "" {
-		return errors.New("`account_token` not specified")
-	}
-	return nil
 }

--- a/exporter/logzioexporter/config.go
+++ b/exporter/logzioexporter/config.go
@@ -22,7 +22,7 @@ import (
 
 // Config contains Logz.io specific configuration such as Account Token, Region, etc.
 type Config struct {
-	configmodels.ExporterSettings `mapstructure:",squash"`
+	configmodels.ExporterSettings 		 `mapstructure:",squash"`
 	Token                         string `mapstructure:"account_token"`           // Your Logz.io Account Token, can be found at https://app.logz.io/#/dashboard/settings/general
 	Region                        string `mapstructure:"region"`                  // Your Logz.io 2-letter region code, can be found at https://docs.logz.io/user-guide/accounts/account-region.html#available-regions
 	CustomEndpoint                string `mapstructure:"custom_endpoint"` 		  // Custom endpoint to ship traces to. Use only for dev and tests. The will override the Region parameter

--- a/exporter/logzioexporter/config.go
+++ b/exporter/logzioexporter/config.go
@@ -23,9 +23,9 @@ import (
 // Config contains Logz.io specific configuration such as Account Token, Region, etc.
 type Config struct {
 	configmodels.ExporterSettings `mapstructure:",squash"`
-	Token                         string `mapstructure:"account_token"`				// Your Logz.io Account Token, can be found at https://app.logz.io/#/dashboard/settings/general
-	Region                        string `mapstructure:"region"`					// Your Logz.io 2-letter region code, can be found at https://docs.logz.io/user-guide/accounts/account-region.html#available-regions
-	CustomListenerAddress         string `mapstructure:"custom_listener_address"`	// Custom endpoint to ship traces to. Use only for dev and tests.
+	Token                         string `mapstructure:"account_token"`           // Your Logz.io Account Token, can be found at https://app.logz.io/#/dashboard/settings/general
+	Region                        string `mapstructure:"region"`                  // Your Logz.io 2-letter region code, can be found at https://docs.logz.io/user-guide/accounts/account-region.html#available-regions
+	CustomEndpoint                string `mapstructure:"custom_endpoint"` 		  // Custom endpoint to ship traces to. Use only for dev and tests. The will override the Region parameter
 }
 
 func (c *Config) validate() error {

--- a/exporter/logzioexporter/exporter.go
+++ b/exporter/logzioexporter/exporter.go
@@ -15,25 +15,8 @@
 package logzioexporter
 
 import (
-	"errors"
-
-	"github.com/logzio/jaeger-logzio/store"
 	"go.opentelemetry.io/collector/component"
 )
-
-// logzioExporter implements an OpenTelemetry trace exporter that exports all spans to Logz.io
-type logzioExporter struct {
-	accountToken string
-	writer       *store.LogzioSpanWriter
-}
-
-func newLogzioExporter(config *Config, params component.ExporterCreateParams) (*logzioExporter, error) {
-
-	if config == nil {
-		return nil, errors.New("exporter config can't be null")
-	}
-	return &logzioExporter{}, nil
-}
 
 func newLogzioTraceExporter(config *Config, params component.ExporterCreateParams) (component.TraceExporter, error) {
 	return nil, nil

